src/main/java/core/basesyntax/service/serviceimpl/DataConverterImpl.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.service.serviceimpl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.DataConverter;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+    public static final String SEPARATOR = ",";
+    public static final int REPORT_START_POSITION_INDEX = 1;
+    public static final int OPERATION_INDEX = 0;
+    public static final int FRUIT_NAME_INDEX = 1;
+    public static final int QUANTITY_INDEX = 2;
+
+    @Override
+    public List<FruitTransaction> convertToTransaction(List<String> fruitInfoList) {
```

Comment:
The method name 'convertToTransaction' could be improved for clarity. Consider renaming it to something more specific to its functionality like 'convertDataToTransactions'.